compiler/crates/relay-compiler/src/artifact_content/content.rs

```diff
@@ -180,22 +178,18 @@ pub fn generate_operation(
     fragment_locations: &FragmentLocations,
 ) -> Result<Vec<u8>, FmtError> {
     let mut request_parameters = build_request_params(normalization_operation);
-    match (id_and_text_hash, &project_config.persist) {
-        (
-            Some(_),
-            Some(PersistConfig::Remote(RemotePersistConfig {
-                include_query_text, ..
-            })),
-        ) if *include_query_text => {
-            request_parameters.id = id_and_text_hash;
-            request_parameters.text = text.clone();
-        }
-        (Some(_), _) => {
-            request_parameters.id = id_and_text_hash;
-        }
-        (None, _) => {
+
```

Comment:
@JingLi1998  resolved a merge conflict here. (Although it looks like something went wrong)